src/backend/utils.ts

```diff
@@ -1070,38 +1061,63 @@ export async function moveOnUnix(
   if (rsyncExists) {
     const origin = install_path + '/'
     logInfo(
-      `moving command: rsync -az --progress ${origin} ${destination} `,
+      `moving command: rsync --archive --compress --no-human-readable --remove-source-files --info=name,progress ${origin} ${destination} `,
       LogPrefix.Backend
     )
     const { code, stderr } = await spawnAsync(
       'rsync',
-      ['-az', '--progress', origin, destination],
+      [
+        '--archive',
+        '--compress',
+        '--no-human-readable',
+        '--remove-source-files',
+        '--info=name,progress',
+        origin,
+        destination
+      ],
       { stdio: 'pipe' },
       (data) => {
-        const split =
-          data
-            .split('\n')
-            .find((d) => d.includes('/') && !d.includes('%'))
-            ?.split('/') || []
-        const file = split.at(-1) || ''
-
-        if (file) {
-          currentFile = file
+        let percent = 0
+        let eta = ''
+        let bytes = '0'
+        // Rsync outputs either the file currently being transferred or a
```

Comment:
one comment on this, when files are small you can get multiple files in the same `data` so it's not either one or the other. I'm printing `data` to the console and I see things like this:

```
api-ms-win-crt-conio-l1-1-0.dll
          20824 100%   36,12kB/s    0:00:00 (xfr#41, ir-chk=1049/1091)
api-ms-win-crt-convert-l1-1-0.dll
          23680 100%   41,07kB/s    0:00:00 (xfr#42, ir-chk=1048/1091)
api-ms-win-crt-environment-l1-1-0.dll
```

(this is all a single `data`)

So in a single string we can have a mix of files and progress updates (I imagine when files are really small).

Reply:
Ugh, and here I was thinking only robocopy did this buffering
Looking at just the last line sounds good though, I'll try that out